### ISSUE TITLE
resource: net::ERR_INSECURE_RESPONSE

### ISSUE THREAD

**Issue by evilword666** [lynckia/licode#918]
I have tested Licode on HTTPS with Chrome
And success to access to local media.
But I get error in console ;
https://xxx.com:8080/socket.io/1/?t=1497286595041  resource: net::ERR_INSECURE_RESPONSE
I can only see my camera but not the one of others that are connected to licode
How to fix it?

**Comment by cracker0dks** [comment]
have you enabled: 
```
config.erizoController.ssl = true
config.erizoController.listen_ssl = true
```
at the licode_config.js ?

**Comment by evilword666** [comment]
yes, but the error is the same:
https://xxx.com:8080/socket.io/1/?t=1497286595041 resource: net::ERR_INSECURE_RESPONSE

**Comment by kekkokk** [comment]
just install a valid certificate for your domain or copy the addrress:
https://xxx.com:8080/socket.io/1/?t=1497378018486
in the bar, accept to load insecure certificate and load basicexample again

**Comment by cracker0dks** [comment]
if you are using a domain you have to set:  config.erizoController.hostname ='my.domain.com'; at the config as well. Maybe post your config if this dont work as well
.
@kekkokk in firefox this might be the problem. Chrome dont require this as I know.

**Comment by kekkokk** [comment]
I have to accept it occasionally, I don't have ever dig into the problem.

**Comment by evilword666** [comment]
Yes, that was the problem. I forgot to put my domain. Now it works. Thank you!
@cracker0dks @kekkokk
